src/main/java/frc/robot/subsystems/intake/IntakeCommandsBuilder.java

```diff
@@ -13,14 +14,29 @@ public IntakeCommandsBuilder(Intake intake) {
 		this.intake = intake;
 	}
 
+	//@formatter:off
 	public Command moveByPower(double power) {
-		return new FunctionalCommand(() -> intake.setPower(power), () -> {}, interrupted -> intake.stop(), () -> false, intake)
-			.withName("Move by power: " + power);
+		return new FunctionalCommand(
+				() -> {},
+				() -> intake.setPower(power),
+				interrupted -> intake.stop(),
+				() -> false,
```

Comment:
we should have isFinished condition if we want to schedule several command one after another